### PR TITLE
NDKilla's localsettings patch (see desc)

### PR DESCRIPTION
**Pull request by NDKilla**
added "bureaucrat" level protection for quantixhl2rpwiki and ironperfectionwiki both owned by NDKilla.
also added namespace aliases for quantixhl2rpwiki

### CODE REVIEW

**Comment by NDKilla** [comment]
Closed outdated pr. Want this still.

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 [hunk 1]
```diff
@@ -1506,6 +1528,12 @@ $wgConf->settings = array(
 		'+metawiki' => array(
 			'createwiki'
 		),
+		'+ironperfectionwiki' => array(
+			'bureaucrat'
```
Comment (Southparkfan) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: Missing comma at end of line

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 [hunk 2]
```diff
@@ -1506,6 +1528,12 @@ $wgConf->settings = array(
...
+		),
+		'+quantixhl2rpwiki' => array(
+			'bureaucrat'
```
Comment (Southparkfan) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: Missing comma at end of line

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 [hunk 3]
```diff
@@ -920,6 +920,12 @@ $wgConf->settings = array(
 			'Server' => NS_SERVER,
 			'Server_talk' => NS_SERVER_TALK,
 		),
+		'quantixhl2rpwiki' => array(
+			'QG' => NS_PROJECT,
+			'QG_Talk' => NS_PROJECT_TALK,
+			'Quantix' => NS_PROJECT,
```
Comment (Southparkfan) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: Are you sure you want these two NOT in wgExtraNamespaces?

Comment (Southparkfan) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: Please define some numbers for these namespaces too: see line 116 of LocalSettings.php.j2 for instructions.

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 (hunk 1 above)
Comment (NDKilla) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: The wiki above, which already exists on meta wiki, doesn't have a comma either. Why do I need a comma if it's the only line and not part of an actual array?

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 (hunk 2 above)
Comment (NDKilla) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: The wiki above, which already exists on meta wiki, doesn't have a comma either. Why do I need a comma if it's the only line and not part of an actual array?

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 (hunk 3 above)
Comment (NDKilla) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: QG is the name of the community
I want QG and Quantix (even tho quantix isn't as short) to redirect to the project namespace, i believe i have these in the right spot to do it.

**Comment by NDKilla** [comment]
ps idk if you saw this just because you're subscribed to the ansible playbook, but I mentioned this p.r. on the Stewards' noticeboard. When you made my wiki (which im editing settings for here; quantixhl2rpwiki) i didn't get founder permissions for it. Soo sometime when you see this could you give me the perms please <3

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 [hunk 4]
```diff
@@ -958,6 +972,12 @@ $wgConf->settings = array(
 			NS_FORUM_ORAIN => true, // NS_FORUM_ORAIN should === NS_FORUM
 			NS_TECH => true,
 		),
+		'+quantixhl2rpwiki' => array(
+			NS_MAIN => true,
+			NS_PROJECT => true,
+			QG => true,
+			Quantix => true,
```
Comment (Southparkfan) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: Wrap this in quotes please.

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 [hunk 5]
```diff
@@ -958,6 +972,12 @@ $wgConf->settings = array(
 			NS_FORUM_ORAIN => true, // NS_FORUM_ORAIN should === NS_FORUM
 			NS_TECH => true,
 		),
+		'+quantixhl2rpwiki' => array(
+			NS_MAIN => true,
+			NS_PROJECT => true,
+			QG => true,
```
Comment (Southparkfan) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: Wrap this in quotes please.

**Pull request merged**